client/src/pages/Profiles/ProfileCard/ProfileCard.tsx

```diff
@@ -0,0 +1,40 @@
+import { Typography, Box, Card } from '@mui/material';
+import useStyles from './useStyles';
+
+export default function ProfileCard({ profile }: any) {
+  const classes = useStyles();
+  const rates = ['$15', '$20', '$25', '$30'];
+  return (
+    <>
+      <Card className={classes.card}>
+        <Box
+          display="flex"
+          flexDirection="column"
+          justifyContent="center"
+          alignItems="center"
+          className={classes.cardContainer}
+        >
+          <img alt="Profile picture" className={classes.profilePic} src={profile.photo}></img>
+          <Typography variant="h6">{profile.name}</Typography>
+          <Typography variant="subtitle2">{profile.blurb}</Typography>
+          <Typography variant="subtitle2">*****</Typography>
+          <Typography variant="subtitle2" className={classes.description}>
+            {profile.description}
+          </Typography>
+          <Box
```

Comment:
We can use a MUI `Divider` component here to separate the footer row from the rest of the card